### PR TITLE
Update all languages plugins to latest VS Code extensions #13521

### PR DESCRIPTION
**Pull request by vrubezhny**
PHP Intelephense Language Support extension is updated to v.1.0.14

Signed-off-by: Victor Rubezhny <vrubezhny@redhat.com>

### What does this PR do?

PHP Intelephense Language Support extension is updated to v.1.0.14

### CODE REVIEW

File: v3/plugins/redhat/php/1.0.14/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/bmewburn/vscode-intelephense
+category: Language
+firstPublicationDate: "2019-06-13"
+spec:
+  containers:
+    - image: "eclipse/che-remote-plugin-php7:next"
+      memoryLimit: "1000Mi"
+  extensions:
+    - https://github.com/che-incubator/vscode-intelephense/releases/download/v1.0.14/bmewburn.vscode-intelephense-client-1.0.14.vsix
```
Comment (ibuziuk) [inline v3/plugins/redhat/php/1.0.14/meta.yaml]: No New line in the end

File: v3/plugins/redhat/php/latest/meta.yaml [hunk 2]
```diff
@@ -9,10 +9,10 @@ type: VS Code extension
...
 category: Language
-firstPublicationDate: "2019-04-16"
+firstPublicationDate: "2019-06-13"
 spec:
   containers:
     - image: "eclipse/che-remote-plugin-php7:next"
       memoryLimit: "1000Mi"
   extensions:
-    - https://github.com/che-incubator/vscode-intelephense/releases/download/v1.0.13/bmewburn.vscode-intelephense-client-1.0.13.vsix
+    - https://github.com/che-incubator/vscode-intelephense/releases/download/v1.0.14/bmewburn.vscode-intelephense-client-1.0.14.vsix
```
Comment (ibuziuk) [inline v3/plugins/redhat/php/latest/meta.yaml]: No New line in the end 
@vrubezhny please take a look at formatting

**Comment by l0rd** [comment]
@vrubezhny what's the status of this PR? Are we ready to merge it? Can you have a look at @ibuziuk comments?

File: v3/plugins/redhat/php/latest/meta.yaml (hunk 2 above)
Comment (nickboldt) [reply v3/plugins/redhat/php/latest/meta.yaml]: does that matter? is the yaml so brittle it breaks w/o a trailing newline?

Comment (amisevsk) [reply v3/plugins/redhat/php/latest/meta.yaml]: No, but `cat` does ;)

**Comment by nickboldt** [comment]
Closing out of date. Replaced by https://github.com/eclipse/che-plugin-registry/pull/349

**Pull request closed without merging**